fuzz/cpio_rdr.c

```diff
@@ -356,7 +365,8 @@ const char *cpio_readentry(CPIO *cpio, size_t *datasize)
         return NULL;
     }
 
-    if (fread(header + 6, cpio->rel_filenameoffset - 6, 1, cpio->file) == 0) {
+    if (fread(header + 6, (size_t)cpio->r_filenameoffset - 6, 1, cpio->file)
```

Comment:
Can `cpio->rel_filenameoffset - 6` underflow?

Reply:
No.  A cpio header is always more than 6 bytes.